.ci/buildci.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+# this script check for 'Circle' in the commit message
+
+MESSAGE=$(git log -1 HEAD --pretty=format:%s)
+
+if [[ "$MESSAGE" == *Circle* ]]; then
```

Comment:
are these changes from a different issue? you need to work on different issues on different branch. These can't be here for this issue